data/json/items/tool_armor.json

```diff
@@ -2860,6 +2860,110 @@
     "flags": [ "WATERPROOF", "STURDY" ],
     "armor": [ { "encumbrance": 20, "coverage": 95, "covers": [ "head" ] } ]
   },
+  {
+    "id": "helmet_sallet",
+    "type": "ARMOR",
+    "category": "armor",
+    "name": { "str": "sallet helm" },
+    "description": "A popular german helmet of the late medieval era, the sallet covers the head and eyes.  Activate it to toggle the visor.",
+    "weight": "1800 g",
+    "volume": "2050 ml",
+    "price": 35000,
+    "price_postapoc": 4500,
+    "use_action": { "type": "transform", "msg": "You raise the sallet visor.", "target": "helmet_sallet_raised", "menu_text": "Raise" },
+    "bashing": 8,
+    "material": [ "steel", "leather" ],
+    "symbol": "s",
+    "looks_like": "helmet_barbute",
+    "color": "light_gray",
+    "armor": [
+      { "covers": [ "head" ], "coverage": 100, "encumbrance": 20 },
+      { "covers": [ "eyes" ], "coverage": 100, "encumbrance": 20 }
+    ],
+    "warmth": 5,
+    "material_thickness": 4,
+    "environmental_protection": 1,
+    "techniques": [ "WBLOCK_1" ],
+    "flags": [ "VARSIZE", "STURDY" ]
+  },
+  {
+    "id": "helmet_sallet_raised",
+    "type": "ARMOR",
+    "category": "armor",
+    "name": { "str": "sallet helm (raised)", "str_pl": "sallet helms (raised)" },
+    "description": "A popular german helmet of the late medieval era, the sallet covers the head and eyes.  Activate it to toggle the visor.",
+    "weight": "1800 g",
+    "volume": "2050 ml",
+    "price": 35000,
+    "price_postapoc": 4500,
+    "use_action": {
+      "type": "transform",
+      "msg": "You lower the sallet visor over your eyes.",
+      "target": "helmet_sallet",
+      "menu_text": "Lower"
+    },
+    "bashing": 8,
+    "material": [ "steel", "leather" ],
+    "symbol": "s",
+    "looks_like": "helmet_barbute",
+    "color": "light_gray",
+    "armor": [ { "covers": [ "head" ], "coverage": 100, "encumbrance": 20 } ],
+    "warmth": 5,
+    "material_thickness": 4,
+    "environmental_protection": 1,
+    "techniques": [ "WBLOCK_1" ],
+    "flags": [ "VARSIZE", "STURDY" ]
+  },
+  {
+    "id": "helmet_armet",
+    "type": "ARMOR",
+    "category": "armor",
+    "name": { "str": "armet helm" },
+    "description": "An iconic knightly helm of the late middle ages.  When you think of a knight's helmet, it looks something like this.  Activate it to toggle the visor.",
+    "weight": "2300 g",
+    "volume": "2050 ml",
+    "price": 55000,
+    "price_postapoc": 5500,
+    "use_action": { "type": "transform", "msg": "You raise the armet visor.", "target": "helmet_armet_raised", "menu_text": "Raise" },
+    "bashing": 8,
+    "material": [ "steel", "leather" ],
+    "symbol": "s",
+    "looks_like": "helmet_barbute",
+    "color": "light_gray",
+    "armor": [
+      { "covers": [ "head" ], "coverage": 100, "encumbrance": 40 },
+      { "covers": [ "eyes" ], "coverage": 100, "encumbrance": 20 },
+      { "covers": [ "mouth" ], "coverage": 100, "encumbrance": 30 }
```

Comment:
```suggestion
      { "covers": [ "mouth" ], "coverage": 85, "encumbrance": 30 }
```

Reply:
85 might be a bit low. I can't imagine anything outside a massive fluke striking someone through the breathing holes in the bevor/visor.

Reply:
Pretty much that, I can see reducing it to like 95 or so, but going off the picture I posted there is almost no way to hit the person in the mouth while wearing an armet.

Reply:
All good. I literally was going off Google pictures and trying to think of the percentage in relation to each other.  The percentage point differences don’t have to be by five points either way but I wanted some discussion

Reply:
yeah, its probably fine either way, I'll just commit the changes you suggested.